content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/install_drivers.rb

```diff
@@ -0,0 +1,34 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class InstallDrivers
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                os = @handle.root['vm'].operating_system
+                is_windows = os && os.product_name =~ /windows/i
```

Comment:
@matobet Minor, but you could use `try` here: `os.try(:product_name) =~ /windows/i`